src/pages/workspace/withFullPolicy.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'underscore';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {useNavigationState} from '@react-navigation/native';
+import getComponentDisplayName from '../../libs/getComponentDisplayName';
+import * as Policy from '../../libs/actions/Policy';
+
+let previousRoute = '';
+
+/*
+ * HOC for loading a full policy. It checks the route params and if current route has a policyID that the previous route did not, it full-loads that policy.
+ */
+export default function (WrappedComponent) {
+    const propTypes = {
+        forwardedRef: PropTypes.func,
+    };
+
+    const defaultProps = {
+        forwardedRef: () => {},
```

Comment:
Can you add a comment here explaining the prop please?

Reply:
done!